### DISCUSSION TITLE
Can I completely revert back to Win 10 Start menu with this?

### DISCUSSION THREAD

**Discussion by Shrimpfren** [valinet/ExplorerPatcher#677] [Q&A] [answered]
If not, will this feature be added, and for the time being, any good alternatives solely for the start menu?

**Comment by valinet** [comment, marked as answer]
> any good alternatives solely for the start menu?
Start11

> will this feature be added
The Microsoft stock menu can be added back by using “StartMenuExperienceHost.exe” from build 22000.51 and the “Statt_ShowClassicMode” registry entry, but it has some serious incompatibilities with the newer system libraries: jump lists do not work, live tiles seem not to work and there is a 1-second delay when dismissing the menu.

I personally am not working on it at the moment. There was someone else that made some progress in this regard, idk what the current status with that is: https://github.com/bbmaster123/10SM

Thanks
